### PR TITLE
[FIX][email_separator][8.0] replacing header method and use of regular expression

### PR DESCRIPTION
**Pull request by Olivier-LAURENT**

### CODE REVIEW

File: email_separator/mail_thread.py [hunk 1]
```diff
@@ -44,10 +43,16 @@ def message_route_check_bounce(self, cr, uid, message, context=None):
...
 
         if bounce_alias in email_to:
-            bounce_re = re.compile('%s%s' % (re.escape(bounce_alias),
-                                             re.UNICODE), BOUNCE_EXPR)
-            if bounce_re:
-                message.set_param('To', message['To'].replace('+', '-', 1))
+            bounce_re = re.compile(
+                r'%s\+(\d+)-?([\w.]+)?-?(\d+)?' % re.escape(bounce_alias),
+                re.UNICODE)
+            if bounce_re.search(email_to):
```
Comment (sbidoul) [inline email_separator/mail_thread.py:51]: Please add a small comment pointing to the original code in mass_mail/mail_thread.py.

**Comment by lmignon** [comment]
:+1:

**Comment by Cedric-Pigeon** [comment]
:+1:

**Comment by laetitia-gangloff** [comment]
:+1:

**Pull request merged**